packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -237,6 +238,30 @@ export class IncomingPayment
     }
     return payment
   }
+
+  public toOpenPaymentsType({
```

Comment:
I expect this can replace a number of existing `toJSON` calls, especially in `routes.ts` files.